compose/ui/ui/src/webCommonW3C/kotlin/androidx/compose/ui/platform/WebTextToolbar.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.platform
+
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.geometry.Rect
+
+class WebTextToolbar : TextToolbar {
```

Comment:
I see that many new source files were added in `webCommonW3C` directory (in ui and foundation modules), but they don't use any W3C APIs and it's possible to keep them in `jsWasmMain` source set, which is our "webMain" without w3c API usages.

Reply:
done